
time time_stamp     # Message 발행 시간 (ros::Time)
int64 signal_bit    # Total Signal Value (bit operation)
